testdata/example-models/nonmem/IOVMM/IOVMM.json

```diff
@@ -0,0 +1,155 @@
+{
+	"run_details": {
+		"version": "7.4.3",
+		"run_start": "Thu Sep  5 14:38:15 EDT 2019",
+		"run_end": "Thu Sep  5 14:38:31 EDT 2019",
+		"estimation_time": 10.94,
+		"covariance_time": 0.57,
+		"function_evaluations": 367,
+		"significant_digits": 3.1,
+		"problem_text": "LEM 10 mixture model and IOV on CL",
+		"estimation_method": [
+			"First Order"
+		],
+		"data_set": "../MixSim.CSV",
+		"number_of_patients": 300,
+		"number_of_obs": 12600,
+		"number_of_data_records": 13500
+	},
+	"run_heuristics": {},
+	"parameters_data": [
+		{
+			"method": "TABLE NO.     1: First Order: Goal Function=MINIMUM VALUE OF OBJECTIVE FUNCTION: Problem=1 Subproblem=0 Superproblem1=0 Iteration1=0 Superproblem2=0 Iteration2=0",
+			"estimates": {
+				"theta": [
+					14.2204,
+					100.996,
+					0.824135,
+					0.988974,
+					1.70925
+				],
+				"omega": [
+					0.0490352,
+					0,
+					0.0482149,
+					0,
+					0,
+					0.0359252,
+					0,
+					0,
+					0,
+					0.0359252,
+					0,
+					0,
+					0,
+					0,
+					0.0359252
+				],
+				"sigma": [
+					0.0458045,
+					0
+				]
+			},
+			"std_err": {},
+			"random_effect_sd": {
+				"omega": [
+					0.221439,
+					0,
+					0.219579,
+					0,
+					0,
+					0.189539,
+					0,
+					0,
+					0,
+					0.189539,
+					0,
+					0,
+					0,
+					0,
+					0.189539
+				],
+				"sigma": [
+					0.21402,
+					0
+				]
+			},
+			"random_effect_sdse": {},
+			"fixed": {
+				"theta": [
+					0,
+					0,
+					0,
+					0,
+					0
+				],
+				"omega": [
+					0,
+					1,
+					0,
+					1,
+					1,
+					0,
+					1,
+					1,
+					1,
+					1,
+					1,
+					1,
+					1,
+					1,
+					1
+				],
+				"sigma": [
+					0,
+					1
+				]
+			}
+		}
+	],
+	"parameter_structures": {
+		"Theta": 6,
+		"Omega": [
+			1,
+			0,
+			2,
+			0,
+			0,
+			3,
+			0,
+			0,
+			0,
+			3,
+			0,
+			0,
+			0,
+			0,
+			3
+		],
+		"Sigma": [
+			1,
+			0,
+			1
+		]
+	},
+	"parameter_names": {
+		"theta": [
+			"1. CL POP 1",
+			"2. V",
+			"3. WT ON CL",
+			"4. WT ON V",
+			"5. CL POP 2",
+			"6. PROB POP 1"
+		]
+	},
+	"ofv": {
+		"ofv": 23157.251036757752,
+		"ofv_no_constant": 17485.751,
+		"ofv_with_constant": 40643.001566747465
+	},
+	"shrinkage_details": {
+		"eta": {},
```

Comment:
hmm why didn't shrinkage get measured

Reply:
I don't think shrinkage requires stderrs to be estimated but i may be wrong

Reply:
There is no data for the shrinkage info in any of the output files. We currently expect strings like this: "ETASHRINKSD(%)" to populate the object.  

The only lines with "shrink" in them are these (from the lst and xml):
IN SHRINKAGE (ETASTYPE):             NO